### ISSUE TITLE
Support for Jinja2 Environment Extensions

### ISSUE THREAD

**Issue by ovanes** [kolypto/j2cli#36]
I am currently running into a problem to use a `do` tag in the template which is going to be processed by j2cli. Please refer to: http://jinja.pocoo.org/docs/2.10/extensions/#expression-statement.

As far as I see these extensions are not supported. I think it'd be a nice feature to support them.

**Comment by kolypto** [comment]
Will you please give me an example of a template, with the input data you use, that give an error?

**Comment by ovanes** [comment]
Desired way to write the code, which needs a `do` expression statement to be enabled:
```
{%- set my_list = [] -%}
{%- for item in data -%}
    {%- for some_other_item in data2 -%}
        {% do my_list.append("'" + item + '.' + some_other_item + "'") %}
    {%- endfor -%}
{%- endfor %}
```

Current workaround:
```
{%- set my_list = [] -%}
{%- for item in data -%}
    {%- for some_other_item in data2 -%}
        {{ my_list.append("'" + item + '.' + some_other_item + "'") | default('', true) }}
    {%- endfor -%}
{%- endfor %}
```

**Comment by kolypto** [comment]
@ovanes , just released 0.3.8, it supports the `do` tag, as well as other extensions (i18n, break/continue). Enjoy! :)

**Comment by ovanes** [comment]
Thanks a lot! Pretty cool and slick support :) I am very happy to have chosen `j2cli`!
